docs/core/domains.rst

```diff
@@ -316,3 +316,29 @@ featurized as normal.
 
     If you really want these entities not to influence action prediction we
     suggest you make the slots with the same name of type ``unfeaturized``.
+
+.. _session_config:
+
+Session configuration
+---------------------
+
+A conversation session represents the dialogue between the assistant and the user.
+Conversation sessions can begin in three ways:
+
+  1. the user begins the conversation with the assistant,
+  2. the user sends their first message after a configurable period of inactivity, or
+  3. a manual session start is triggered with the ``/session_start`` intent message.
+
+You can define the period of inactivity after which a new conversation
+session is triggered in the domain under the ``session_config`` key.
+``session_expiration_time`` defines the time of inactivity in minutes after which a
+new session will begin. ``carry_over_slots_to_new_session`` determines whether
+existing set slots should be carried over to new sessions.
```

Comment:
Can you add a little yaml example like above? You can use it to show the defaults as well. (with a comment that 0 means infinite sessions)

knowing the defaults here as well as on the tracker page would be dope

Reply:
maybe it makes the most sense to stick in an example after line 333, since it talks about where to put the info, and then the following ones explain the keys

Reply:
👍 added here and to the tracker docs